_events/sr2019/london-tech-day-december.md

```diff
@@ -16,32 +16,8 @@ limited to six teams, so be sure to [let us know][teams-contact] if you'd like
 to attend. For this Tech Day, please express your interest by Wednesday 12th
 December.
 
-**Please bring your laptops!**
+~~Please bring your laptops!~~
```

Comment:
Why are we changing this to be crossed out?

Reply:
It doesn't make sense to leave it in place, but this felt a better solution than removing it. Happy to remove it if you think that's better.

Reply:
I'd argue leaving it as-is, it feels unnecessary to change it

Reply:
My concern was that leaving it as it was could be confusing as it somewhat implies the event is going ahead.

Reply:
Removed entirely in https://github.com/srobo/website/pull/121/commits/9e05007c630176ea45eddc28c96fe59a80d61ec2 as that seems simplest.